docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source the following file, you can use bash/zsh shell functions which provide added utility.
+
+.. code-block:: shell
+
+    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
```

Comment:
what I found so far is follows:
```
 os.path.abspath(os.path.join(os.path.dirname(inspect.getfile(catkin_tools)), '../../../../share/catkin_tools'))
```
```
os.path.abspath(os.path.join(imp.find_module('catkin_tools')[1], '../../../../share/catkin'))
```
if we do not have other solution, we have to accept this ugly expresson, or put `catkin_shell_verbs.bash` under `dist-packages`